aspnetcore/performance/response-compression/sample/License.txt

```diff
@@ -0,0 +1,12 @@
+Copyright (c) .NET Foundation. All rights reserved.

```

Comment:
> nice to be consistent

I've heard that somewhere before. :smile:

@Rick-Anderson @tdykstra Feel free to add this to the *Rex Consistency PR Series*.